tests/test_schemas.py

```diff
@@ -54,40 +87,47 @@ def load_as_dataset(filepath):
     return iati.Dataset(xml_str)
 
 
-@pytest.mark.parametrize('filepath', list_xml_files_recursively('tests/activity-tests/should-pass/') +  # Legacy tests
-                         list_xml_files_recursively('tests/should-pass/'))  # Tests in new format
-def test_pass_files(activity_schema, filepath):
-    """Check that all 'should-pass' test files are XML and pass Schema validation."""
+@pytest.mark.parametrize('filepath', list_xml_files_recursively('tests/activity-tests/should-pass/') +  # Legacy activity test cases
+                         list_xml_files_recursively('tests/should-pass/iati-activities/') +  # Activity test cases in new format
+                         list_xml_files_recursively('tests/organisation-tests/should-pass/') +  # Legacy organisaion test cases
+                         list_xml_files_recursively('tests/should-pass/iati-organisations/'))  # Organisation test cases in new format
+def test_pass_files(schema_factory, filepath):
+    """Check that all activity and organisation schema 'should-pass' test files are XML and pass Schema validation."""
     dataset = load_as_dataset(filepath)
+    schema = schema_factory.get_correct_schema_for_filepath(filepath)
 
     assert iati.validator.is_xml(dataset)
-    assert iati.validator.is_iati_xml(dataset, activity_schema)
+    assert iati.validator.is_iati_xml(dataset, schema)
+
```

Comment:
Should this be 2 blank lines? Not sure but it looks odd...

Reply:
According to the linter, there should be two blank lines between function definitions.

Reply:
Ah, there is now. Good good!